### ISSUE TITLE
app.py AttributeError

### ISSUE THREAD

**Issue by lhy0807** [kevincon/dotaml#6]
Hi!
Your project is amazing! I am very interested in your project and want to try it or even improve it. I am facing problems when I run app.py. It keep telling me that `AttributeError: 'KNeighborsClassifier' object has no attribute 'n_jobs'` or `DeprecationWarning: Passing 1d arrays as data is deprecated in 0.17 and will raise ValueError in 0.19. Reshape your data either using X.reshape(-1, 1) if your data has a single feature or X.reshape(1, -1) if it contains a single sample.
  DeprecationWarning)`

Is this because we are using the different versions of sklearn libraries?

**Comment by kevincon** [comment]
> Is this because we are using the different versions of sklearn libraries?

Yes I think so. It sounds like you have a version of sklearn that is at least 0.17 (`Passing ld arrays as data is deprecated in 0.17...`). The `requirements.txt` file in this repo wants 0.14.1, so it may be easier to just use virtualenv with this repo so you get the same versions of the requirements that we used originally.
